core/modules/system/system.menu.inc

```diff
@@ -0,0 +1,332 @@
+<?php
+
+/**
+ * Implements hook_block_configure().
+ */
+function _system_block_configure($delta = '', $settings = array()) {
+  // Create a pseudo form state.
+  // Merge in the default configuration.
+  $settings += system_menu_default_config();
+  $settings['delta'] = $delta;
+  return system_menu_configure_form($settings);
+}
+
+/**
+ * The default menu block configuration.
+ *
+ * @return array
+ */
+function system_menu_default_config() {
+  return array(
+    'parent' => 'main-menu:0',
+    'title_link' => 0,
+    'admin_title' => '',
+    'level' => 1,
+    'follow' => 0,
+    'depth' => 0,
+    'expanded' => 0,
+    'sort' => 0,
+    'menu_name' => 'main-menu',
+    'parent_mlid' => 0,
```

Comment:
These options can be trimmed down substantially. Now that we're only providing two options, I think `level` and `depth` are the only necessary options?